### DISCUSSION TITLE
Does 0.14.8 support Prisma 2.29.1 ?

### DISCUSSION THREAD

**Discussion by omar-dulaimi** [MichalLytek/typegraphql-prisma#165] [Q&A]
Hello,

I wanted to try out the named constrasints preview feature, but saw the typegraphql prisma version is behind a patch release from prisma side.

So, does typegraphql prisma 0.14.8 support 2.29.1?

**Comment by MichalLytek** [comment]
Yes.
https://github.com/MichalLytek/typegraphql-prisma/releases/tag/v0.14.8

**Reply by MichalLytek** [reply]
Next time try to install `2.29.1` and check by yourself, instead of engaging others in answering you 😉

**Reply by omar-dulaimi** [reply]
Well, I remembered a note here saying that it works with exact versions of prisma, hence why I asked. If it wasn't clear I wouldn't have asked, and to be completely honest you're not friendly enough/welcoming to be asked according to my experience with you. I appreciate and aware that you're supporting the library on your own mainly and provide quick responses sometimes(but also ignore others).

**Reply by MichalLytek** [reply]
![image](https://user-images.githubusercontent.com/10618781/130348108-ddbae694-5f73-4f9c-9e10-c3089ed38b7f.png)

https://jubianchi.github.io/semver-check/#/~2.29/2.29.1
